spec/timezone.html

```diff
@@ -483,6 +504,25 @@ <h1>
         When the input represents a skipped local time at a positive time zone transition (e.g. when the daylight saving time starts or the time zone offset is increased due to a time zone rule change), the returned List will be empty.
         Otherwise, the returned List will have one element.
       </p>
+      <p>
+        The minimum implementation of GetIANATimeZoneEpochValue for ECMAScript implementations that do not include local political rules for any time zones, performs the following steps when called:
+      </p>
+      <emu-alg>
+        1. Assert: _timeZoneIdentifier_ is *"UTC"*.
```

Comment:
I think this kind of assertion should be present in all the "minimum" implementations that accept input.